### PR TITLE
Nikshay to eNikshay migration

### PR DESCRIPTION
**Pull request by nickpell**
For architecture review

@czue @proteusvacuum

### CODE REVIEW

File: custom/enikshay/nikshay_datamigration/models.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+class PatientDetail(models.Model):
+    PregId = models.CharField(max_length=255, primary_key=True)
+    Stocode = models.CharField(max_length=255, null=True)
+    Dtocode = models.CharField(max_length=255, null=True)
+    Tbunitcode = models.IntegerField(null=True)
+    pname = models.CharField(max_length=255, null=True)
+    pgender = models.CharField(max_length=255)
+    page = models.IntegerField(null=True)
+    poccupation = models.IntegerField(null=True)
+    paadharno = models.CharField(max_length=255, null=True) # big ints (scientific notation) and nulls. requires some formatting
```
Comment (dimagimon) [inline custom/enikshay/nikshay_datamigration/models.py]: E261 at least two spaces before inline comment
E501 line too long (128 > 115 characters)

File: custom/enikshay/nikshay_datamigration/models.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    dcpulmunorydet = models.CharField(max_length=255, null=True)
+    dotname = models.CharField(max_length=255, null=True)
+    dotdesignation = models.CharField(max_length=255, null=True)
+    dotmob = models.CharField(max_length=255, null=True)
+    dotlandline = models.CharField(max_length=255, null=True)
+    dotpType = models.IntegerField()
+    dotcenter = models.CharField(max_length=255, null=True)
+    PHI = models.IntegerField()
+    dotmoname = models.CharField(max_length=255, null=True)
+    dotmosignDate = models.CharField(max_length=255, null=True)  # datetimes, look like they're all midnight. also have a bunch of 1/1/1990
```
Comment (dimagimon) [inline custom/enikshay/nikshay_datamigration/models.py]: E501 line too long (139 > 115 characters)

File: custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                domain='enikshay-np',
+                case_id=make_uuid(),
+                case_name=make_uuid(),
+                case_properties={
+                    'nikshay_id': outcome.PatientId.PregId,
+                },
+                case_type='occurrence',
+            )
+
+class Command(BaseCommand):
```
Comment (dimagimon) [inline custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py]: E302 expected 2 blank lines, found 1

File: custom/enikshay/nikshay_datamigration/models.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
+from django.db import models
+
+
+class PatientDetail(models.Model):
+    PregId = models.CharField(max_length=255, primary_key=True) # need to remove trailing whitespace in Excel
```
Comment (dimagimon) [inline custom/enikshay/nikshay_datamigration/models.py]: E261 at least two spaces before inline comment

**Comment by nickpell** [comment]
Made updates based on conversation and added an example of using a foreign key to access another table.

@czue @proteusvacuum

**Pull request closed without merging**